### DISCUSSION TITLE
Motion detect slow moving objects

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#9999] [General]
I have a requiremet to detect motion on slow moving critters in the basement (snake). I only want to detect motion if it persists over say 3 seconds.
I scoured the documentation and forums but cannot figure out a way or if this is possible.
Thank you

**Comment by NickM-27** [comment]
the best way this could be done would be setting the motion_off_delay to 1 and then checking if motion lasts for 3 seconds
